picasso/src/main/java/com/squareup/picasso3/Request.java

```diff
@@ -573,6 +579,24 @@ public Builder priority(@NonNull Priority priority) {
       return this;
     }
 
+    @NonNull
+    public Builder asBitmap() {
```

Comment:
This is to make it easier to specify the exact image decoder, avoiding checking all of the default ones. This is completely optional, so no code changes are required.

Usage:

```java
Picasso.get()
  .load(myImageUrl)
  .asBitmap()
  .into(myImageView)l
```

Reply:
umm, that's the `RequestCreator` one :)

Reply:
This is fine for now, but I want to nuke this whole API from orbit. I'd like requests to be separate from how they're fulfilled.